### PR TITLE
Fix #17

### LINKED ISSUE #17: since v0.30 log level is not fatal anymore
Logging is disabled by default in fastify, however I think in the CLI we should default it to info, and maybe throw in `pino-colada` as well (maybe), and __definitely remove the `console.log` statements__.

cc @allevo @delvedor

### PR DESCRIPTION
**Pull request by allevo**

### CODE REVIEW

**Comment by allevo** [comment]
@mcollina  `npm install` fails on node 4. Job restarted

**Comment by allevo** [comment]
@mcollina I'm not able to merge on this repo. Anyway if you want to wait for @delvedor it's ok for me. I'm not in hurry.

File: cli.js [hunk 1]
```diff
@@ -120,7 +120,8 @@ function runFastify (opts) {
     if (typeof address === 'object') {
       address = `http://localhost:${fastify.server.address().port}`
     }
-    console.log(`Server listening on ${address}`)
+    fastify.log.info(`Server listening on ${address}`)
+    console.log()
```
Comment (mcollina) [inline cli.js]: why are we keeping an empty `console.log` here?

File: cli.js [hunk 2]
```diff
@@ -120,7 +120,8 @@ function runFastify (opts) {
     if (typeof address === 'object') {
       address = `http://localhost:${fastify.server.address().port}`
     }
-    console.log(`Server listening on ${address}`)
+    fastify.log.info(`Server listening on ${address}`)
```
Comment (mcollina) [inline cli.js]: should we move this log line to fastify itself?

Comment (allevo) [reply cli.js]: It's ok for me

**Comment by mcollina** [comment]
@allevo can you send a PR to fastify to add that logging statement?

**Pull request merged**